### DISCUSSION TITLE
One button startup for Spotify playing playlist on soundbar ?

### DISCUSSION THREAD

**Discussion by Soiski71** [PiotrMachowski/Home-Assistant-custom-components-SmartThings-Soundbar#73] [Q&A] [answered]
I'm trying to generate one button to start spotify (and play certain playlist) on this soundbar. If I start spotify from its web browser and "jump start it" there first then I can use the button in HA, but after approx 5 minutes it goes to Idle ( if nothing is playing ) and then the button (script) will not start it.
Have you perhaps managed to get this working ?

I believe this is actually a Spotify Add-on bug...

**Comment by PiotrMachowski** [comment]
Personally I don't use Spotify on my soundbar, I use Chromecast Audio connected to it instead. You might try to use [spotcast](https://github.com/fondberg/spotcast) to start Spotify

**Reply by Soiski71** [reply]
I actually tested it already and seems that the Soundbar is not recognized as Entity in the selection. Only official Chromecast devices shown. Can you see your device. I have HW Q800D soundbar.

![image](https://github.com/user-attachments/assets/4c597204-c729-44ed-9004-1d5ca13f64df)

**Reply by PiotrMachowski** [reply]
You have to do it using Spotify device ID
https://github.com/fondberg/spotcast?tab=readme-ov-file#start-playback-on-spotify-connect-device

Starting by entity ID works just for Chromecast devices

**Reply by Soiski71** [reply, marked as answer]
Thanks that worked like charm !
